image-swipe.ios.ts

```diff
@@ -299,9 +299,12 @@ export class ImageSwipe extends ImageSwipeBase {
         const scrollView: UIScrollView = this.ios;
         const width = utils.layout.toDeviceIndependentPixels(this.getMeasuredWidth());
         const height = utils.layout.toDeviceIndependentPixels(this.getMeasuredHeight());
+        const safeAreaInsets = this.getSafeAreaInsets();
+        const insetAllowance = utils.layout.toDeviceIndependentPixels(safeAreaInsets.left) * 2;
```

Comment:
Prefer this to be 
```ts
const insetAllowance = utils.layout.toDeviceIndependentPixels(safeAreaInsets.left + safeAreaInsets.right)
```
So it is more clear.

Reply:
Good suggestion. This is pushed.